src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -121,8 +91,39 @@ $schedule-btn--height: 51px;
     }
   }
 
-  &__bookmarked-toggle {
+  &__bookmarkedToggle {
     justify-content: left;
     padding: $spacing--sm 0px;
   }
+
+  &__downloadButtons {
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    width: 100%;
+    z-index: z(navbar-schedule-download);
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    background-color: opaque-black(0.9);
+    height: $schedule-btn-height;
+    padding-left: $navbar-schedule-padding;
+  }
+
+  // @debt: overwrites default btn btn-primary styles
+  &__downloadScheduleBtn {
```

Comment:
```suggestion
  &__download-schedule-button {
```

Reply:
fixed in 2c16847b0928ba957a737453f7b445db887dda23